bundles/org.openhab.binding.bambulab/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,155 @@
+				<description>Speed of the second large cooling fan (RPM).</description>
+			</channel>
+			<channel id="heatBreakFanSpeed" typeId="number-channel">
+				<label>Heat Break Fan Speed</label>
+				<description>Speed of the heat break cooling fan (RPM).</description>
+			</channel>
+			<channel id="layerNum" typeId="number-channel">
+				<label>Current Layer Number</label>
+				<description>Current layer being printed.</description>
+			</channel>
+			<channel id="speedLevel" typeId="number-channel">
+				<label>Print Speed Level</label>
+				<description>Current speed setting of the print job.</description>
+			</channel>
+			<channel id="timeLaps" typeId="boolean-channel">
+				<label>Timelapse Enabled</label>
+				<description>Indicates whether timelapse recording is enabled.</description>
+			</channel>
+			<channel id="useAms" typeId="boolean-channel">
+				<label>AMS System in Use</label>
+				<description>Indicates whether the Automatic Material System (AMS) is active.</description>
+			</channel>
+			<channel id="vibrationCalibration" typeId="boolean-channel">
+				<label>Vibration Calibration</label>
+				<description>Indicates whether vibration calibration has been performed.</description>
+			</channel>
+
+			<!-- command channels -->
+			<channel id="ledChamber" typeId="on-off-command-channel">
+				<label>Chamber LED</label>
+				<description>Controls the LED lighting inside the printer chamber.</description>
+			</channel>
+			<channel id="ledWork" typeId="on-off-command-channel">
+				<label>Work Area LED</label>
+				<description>Controls the LED lighting for the work area.</description>
+			</channel>
+		</channels>
+
+		<representation-property>serial</representation-property>
+		<config-description>
+			<parameter name="serial" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Unique serial number of the printer.</description>
+			</parameter>
+
+			<parameter name="scheme" type="text" required="false">
+				<label>Scheme</label>
+				<description>URI scheme.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="hostname" type="text" required="true">
+				<label>Hostname</label>
+				<description>IP address of the printer or `us.mqtt.bambulab.com` for cloud mode.</description>
+			</parameter>
+
+			<parameter name="port" type="integer" required="false">
+				<label>Port</label>
+				<description>URI port.</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="username" type="text">
+				<label>Username</label>
+				<description>`bblp` for local mode or your Bambu Lab user (starts with `u_`).</description>
+				<advanced>true</advanced>
+			</parameter>
+
+			<parameter name="accessCode" type="text" required="true">
+				<label>Access Code</label>
+				<description>Access code of the printer. The method of obtaining it differs for local and cloud modes.
+				</description>
+				<context>password</context>
```

Comment:
Is it intentional that the username is advanced and the accesscode is not? 
The acces code is only needed for cloud or not and all cloud related settings are  advanced?

Reply:
> Is it intentional that the username is advanced and the accesscode is not?
yes, for local mode username is always `bblp` so user do not need to set it up at all

> The acces code is only needed for cloud or not and all cloud related settings are advanced?
access code is required for both local and cloud